<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserWindow</class>
 <widget class="QDialog" name="ParserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>

    <width>721</width>
    <height>518</height>

   </rect>
  </property>
  <property name="windowTitle">
   <string>Parser</string>
  </property>
  <widget class="QPushButton" name="Prettifybtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Prettify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Minifybtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Minify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Decompressbtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Decompress</string>
   </property>
  </widget>
  <widget class="QPushButton" name="correctbtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Correct</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Detectbtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JsonExpButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Export As Json</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Compressbtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Compress</string>
   </property>
  </widget>
  <widget class="QPushButton" name="XMLExpButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Export As XML</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>60</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>280</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
